group_vars/all.yml.sample

```diff
@@ -478,13 +478,22 @@ dummy:
 #rgw_multisite: false
 
 # The following Multi-site related variables should be set by the user.
+#
```

Comment:
you need to edit ./roles/ceph-defaults/defaults/main.yml and run the generate sample script instead of editing this file directly.